$NetBSD: patch-prosody.cfg.lua.dist,v 1.3 2025/10/15 10:34:42 sborrill Exp $

Set the logging paths
Set up the pid files
Set the location of the socket for prosodyctl

--- prosody.cfg.lua.dist.orig	2022-03-08 12:34:39.594708341 +0000
+++ prosody.cfg.lua.dist
@@ -183,8 +183,8 @@ archive_expires_after = "1w" -- Remove a
 -- Logging configuration
 -- For advanced logging see https://prosody.im/doc/logging
 log = {
-	info = "prosody.log"; -- Change 'info' to 'debug' for verbose logging
-	error = "prosody.err";
+	info = "@PROSODY_LOG@/prosody.log"; -- Change 'info' to 'debug' for verbose logging
+	error = "@PROSODY_LOG@/prosody.err";
 	-- "*syslog"; -- Uncomment this for logging to syslog
 	-- "*console"; -- Log to the console, useful for debugging when running in the foreground
 }
@@ -205,6 +205,12 @@ log = {
 -- Location of directory to find certificates in (relative to main config file):
 certificates = "certs"
 
+-- Enable PID file by default
+pidfile = "@PROSODY_RUN@/prosody.pid";
+
+-- Configure location of admin socket
+admin_socket = "@PROSODY_RUN@/prosody.sock";
+
 ----------- Virtual hosts -----------
 -- You need to add a VirtualHost entry for each domain you wish Prosody to serve.
 -- Settings under each VirtualHost entry apply *only* to that host.
